### PR TITLE
Support expireCache in amp-consent prompt action

### LINKED ISSUE #28684: Consent: Support the reprompt API to clear stored consent
The `prompt` action will take arg `expireCache=true` (default to false).
When set to `true`, AMP will set dirtyBit to the stored consent information and clear value on next user visit.

### PR DESCRIPTION
**Pull request by zhouyx**
Closes #28684

### CODE REVIEW

File: test/manual/amp-consent/amp-consent-reprompt.html [hunk 1]
```diff
@@ -0,0 +1,218 @@
+<!doctype html>
```
Comment (micajuine-ho) [inline test/manual/amp-consent/amp-consent-reprompt.html:1]: Thanks for the manual test! Maybe we could make this test-page include buttons to test all interactions?

Comment (zhouyx) [reply test/manual/amp-consent/amp-consent-reprompt.html:1]: I think we do have all the actions in this example. We have `accept` `reject` and `dismiss` in the UI. And the two versions of `prompt()` and `prompt(expireCache=true)`.

Any other interactions to include?

Comment (micajuine-ho) [reply test/manual/amp-consent/amp-consent-reprompt.html:1]: Oops, guess I didn't see that. Then maybe it's worth renaming the file to `amp-consent-actions.html` or something similar.

Comment (zhouyx) [reply test/manual/amp-consent/amp-consent-reprompt.html]: sg. done

**Pull request merged**